zap/tests/conftest.py

```diff
@@ -1,6 +1,6 @@
 import pytest
 from itertools import chain
-
+from brownie import accounts, y3CrvZapper, CurveBackzapper
```

Comment:
I think I can remove this

Reply:
Yes, use dependency injection via arguments with all the fixtures.